zgw/portaaltaak/src/main/kotlin/com/ritense/portaaltaak/PortaaltaakPlugin.kt

```diff
@@ -91,6 +93,7 @@ class PortaaltaakPlugin(
         @PluginActionProperty receiver: TaakReceiver,
         @PluginActionProperty identificationKey: String?,
         @PluginActionProperty identificationValue: String?,
+        @PluginActionProperty verloopDurationInDays: Long?,
```

Comment:
This doesn't break when an implementation already has this plugin-action configured?

Reply:
The tests already covered a null argument situation. I added another test to verify that the framework will pass in a null if the property is missing from the configuration. So this shouldn't break.